apps/web/src/config/index.ts

```diff
@@ -94,6 +94,9 @@ export const CLERK_PUBLISHABLE_KEY =
 export const IS_UNDER_DASHBOARD =
   window._env_.REACT_IS_UNDER_DASHBOARD || process.env.REACT_IS_UNDER_DASHBOARD || 'false';
 
+export const WEB_APP_URL =
+  window._env_.REACT_APP_WEB_APP_URL || process.env.REACT_APP_WEB_APP_URL || window.location.origin;
```

Comment:
when the web application is embedded in the new dashboard we need to know the original web application URL so we can redirect it properly to the login page